{"name": "test_div[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.1 == 1", "trace": "self = <calculator.testcase.test_calc_case.TestCalc object at 0x000001B3853B83D0>, get_calc = <calculator.python_code.calc.Calculator object at 0x000001B385325A00>\nget_div_datas = [2.2, 2, 1]\n\n    @allure.story(\"除法测试案例\")\n    @pytest.mark.run(order=4)\n    # @pytest.mark.fourth\n    def test_div(self, get_calc, get_div_datas,):\n        \"\"\"\n        get_calc实例化计算器类\n        get_div_datas:获取测试数据\n        \"\"\"\n        with allure.step(\"计算两个数的相除结果\"):\n            # 获取yaml文件内参数\n            result = get_calc.div(get_div_datas[0], get_div_datas[1])\n            # 如果相加数为float型 ，保留两位小数\n        if isinstance(result, float):\n            result = round(result, 2)\n            # 结果添加断言\n>       assert result == get_div_datas[2]\nE       assert 1.1 == 1\n\ntest_calc_case.py:47: AssertionError"}, "description": "\n        get_calc实例化计算器类\n        get_div_datas:获取测试数据\n        ", "steps": [{"name": "计算两个数的相除结果", "status": "passed", "start": 1612516764593, "stop": 1612516764593}], "attachments": [{"name": "stdout", "source": "5c0a4f0e-a156-4f5f-8f6d-328d322c4ca6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_div_datas", "value": "[2.2, 2, 1]"}], "start": 1612516764593, "stop": 1612516764593, "uuid": "54b21f1d-2f1c-493c-aef7-be8cd9846153", "historyId": "24d802005b79b777f080e42096a4e473", "testCaseId": "fe9a846428a25de930497f4fe00c4783", "fullName": "test_calc_case.TestCalc#test_div", "labels": [{"name": "feature", "value": "测试计算器类"}, {"name": "story", "value": "除法测试案例"}, {"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_calc_case"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-DH8SI2H"}, {"name": "thread", "value": "32452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_case"}]}